doc/concepts/interconnectivity.rst

```diff
@@ -51,11 +57,14 @@ A recipe lets you define which sites are connected to which.
 
       Similarly to `Connections`, Gap Junctions in Arbor are defined in two steps:
 
-      1. Create a **gap junction site** on two separate cells as part of their
+      1. Create labeled **gap junction sites** on two separate cells as part of their
          :ref:`cell descriptions <modelcelldesc>` in the :ref:`recipe <modelrecipe>`.
-      2. Declare the Gap Junction connections in the recipe: between a peer and local **gap junction site**
-         identified using :gen:`cell_member` and a conductance in μS. Two of these connections are needed,
-         on each of the peer and local cells.
+         Each labeled group of gap junctions may contain multiple items on possibly multiple locations
+         on the cell.
+      2. Declare the Gap Junction connections in the recipe *on the local cell*: from a peer **gap junction site**
+         identified using a :gen:`global_label`; to a local **gap junction site** identified using a
+         :gen:`local_label` (:gen:`gid` of the site is implicitly known); and a conductance in μS.
```

Comment:
"implicitly known": can you add that it is known by way of the ``gid`` of the recipe function call?